phpBB/styles/prosilver/theme/content.css

```diff
@@ -491,18 +491,17 @@ blockquote cite > div {
 
 /* Code block */
 .codebox {
-	padding: 3px;
 	border: 1px solid transparent;
 	font-size: 1em;
 	margin: 1em 0 1.2em 0;
-	overflow-x: scroll;
 	word-wrap: normal;
 }
 
 .codebox p {
 	text-transform: uppercase;
 	border-bottom: 1px solid transparent;
-	margin-bottom: 3px;
+	margin-bottom: 0;
+	padding: 3px 3px 0 3px;
```

Comment:
Indeed they are ugly.  It would be nice if we could have custom scrollbars (jQuery UI?) in 3.3 for the code box, the notifications dropdown, the topic review, the jumpbox etc. and make them look uniform across different platforms.

(P.S.: I haven't really changed the resulting padding - I just applied it elsewhere to get the same effect.)